context/csv2js.py

```diff
@@ -70,52 +77,93 @@ def read_csvs(csvs, primary_key):
     }
 
 
-def get_column(column, list_of_dicts):
+def get_raw_column(column, list_of_dicts):
+    '''
+    >>> get_raw_column('a', [{'a':'1'}])
+    ['1']
+    '''
+    includes_nones = [d.get(column) for d in list_of_dicts]
+    return [val for val in includes_nones if val is not None]
```

Comment:
Could be reduced to something like:
`filter(None, map(lambda d: d.get(column), list_of_dicts))` but thats probably less clear

Reply:
`filer(None, ...)` would remove false-ish values, and in this context I wanted to be sure I was only removing `None`s.